src/Components/CardForm/CardForm.tsx

```diff
@@ -1,13 +1,41 @@
-import React, { FC, useState, ChangeEvent } from 'react';
+import React, { FC, useState, ChangeEvent, useEffect } from 'react';
+import { Link } from 'react-router-dom';
 import './CardForm.css';
 
-const CardForm: FC = () => {
+interface Props {
+    addCard:(card: object) => void
+}
+
```

Comment:
Lets use this as an opportunity to practice talking about our new tech - would some of you be willing to tell me what this code is doing?